sktime/benchmarking/benchmarks.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+"""Benchmarking interface for use with sktime objects.
+
+Wraps kotsu benchmarking interface.
+"""
+from typing import Callable, Optional, Type, Union
+
+import kotsu
```

Comment:
I would suggest to move the import *inside* the constructor and inside the `run`.

To complete soft dependency encapsulation, kindly add a `_check_soft_dependencies` at the start of the `__init__`.

Reply:
Done in [4e18ac4](https://github.com/alan-turing-institute/sktime/pull/2977/commits/4e18ac46611ae0cea5aba52c5afbc753c105dc6b)